lib/views/home_content/feed/feed_sort_option_chips.dart

```diff
@@ -0,0 +1,42 @@
+import "package:flutter/material.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/services/sorting/post_sort_option.dart";
+import "package:proxima/viewmodels/feed_sort_options_view_model.dart";
+
+/// This widget is the list of sort options for the timeline.
+class FeedSortOptionChips extends ConsumerWidget {
+  static final optionChipKey = Map.fromIterables(
+    PostSortOption.values,
+    PostSortOption.values.map((option) => Key("optionChip${option.name}")),
+  );
+
+  const FeedSortOptionChips({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final selectedOption = ref.watch(feedSortOptionsProvider);
+
+    final sortOptions = PostSortOption.values.map((sortOption) {
+      return ChoiceChip(
+        key: optionChipKey[sortOption],
+        selected: sortOption == selectedOption,
+        label: Text(sortOption.name),
+        onSelected: (value) {
+          if (value) {
+            ref
+                .read(feedSortOptionsProvider.notifier)
+                .setSortOption(sortOption);
+          }
+        },
+      );
+    }).toList();
```

Comment:
Good job with the UI :+1: 
Small suggestion, I believe that we can use the arrow synthax here.

```suggestion
        final sortOptions = PostSortOption.values
        .map(
          (sortOption) => ChoiceChip(
            key: optionChipKey[sortOption],
            selected: sortOption == selectedOption,
            label: Text(sortOption.name),
            onSelected: (value) {
              if (value) {
                ref
                    .read(feedSortOptionsProvider.notifier)
                    .setSortOption(sortOption);
              }
            },
          ),
        )
        .toList();
```

Reply:
Fixed in b0dec4962087d323e1efed7f1238fd02a8cf7235.